src/main/java/com/revature/model/Skill.java

```diff
@@ -1,120 +1,112 @@
 package com.revature.model;
-
 import java.util.HashSet;
+import java.util.Objects;
 import java.util.Set;
-
 import javax.persistence.CascadeType;
 import javax.persistence.Column;
 import javax.persistence.Entity;
 import javax.persistence.FetchType;
 import javax.persistence.GeneratedValue;
 import javax.persistence.GenerationType;
 import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
 import javax.persistence.ManyToMany;
 import javax.persistence.Table;
-
+import com.fasterxml.jackson.annotation.JsonIdentityInfo;
 import com.fasterxml.jackson.annotation.JsonIgnore;
-
+import com.fasterxml.jackson.annotation.ObjectIdGenerators;
 /**
  * An object representation of our Skill model.
  * 
  * @author Davin Merry
  */
 @Entity
 @Table(name = "skills")
-public class Skill {
 
+public class Skill {
 	@Id
 	@Column(name = "skill_id")
 	@GeneratedValue(strategy = GenerationType.IDENTITY)
 	/** An integer that uniquely identifies this skill. */
 	private int id;
-
+	

```

Comment:
Don't make a change that just adds a tab character to a blank line.

Reply:
Changes are made as per suggestion